### PR TITLE
tbl2asn: new package at 20200301

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

**Comment by adamjstewart** [comment]
Let's add Spack support for decompressing `.gz` tarballs.

**Comment by adamjstewart** [comment]
Hmm, this should already work: https://github.com/spack/spack/blob/develop/lib/spack/spack/util/compression.py#L32

What problem did you experience? Was it on an old version of Spack?

**Comment by codeandkey** [comment]
Fixed; Spack was correctly extracting the binary file but to a different location than I expected. Modified the package accordingly.

File: var/spack/repos/builtin/packages/tbl2asn/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from os import chmod
+
+
+class Tbl2asn(Package):
+    """Tbl2asn is a command-line program that automates the creation of
+       sequence records for submission to GenBank."""
+
+    homepage = "https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/"
+
+    version('20200301', sha256='7cc1119d3cfcbbffdbd4ecf33cef8bbdd44fc5625c72976bee08b1157625377e')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/tbl2asn/package.py]: I would call the version 2020-03-01

Comment (codeandkey) [reply var/spack/repos/builtin/packages/tbl2asn/package.py]: updated

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**